js/src/common/components/Page.js

```diff
@@ -34,6 +34,8 @@ export default class Page extends Component {
    */
   onNewRoute() {
     app.current.set('routeName', this.attrs.routeName);
+    //abort all previous requests to prevent bugs
+    for (let request in app.activeRequests) request.abort();
```

Comment:
When the request is aborted, how does that affect the `.then` handling above? Would that count as an error, or would neither of those 2 blocks run? I wasn't able to find that bit in Mithril documentation

Reply:
Tested that neither will run